packages/babel-parser/src/parser/expression.js

```diff
@@ -1754,7 +1754,7 @@ export default class ExpressionParser extends LValParser {
       }
       // get PropertyName[?Yield, ?Await] () { FunctionBody[~Yield, ~Await] }
       // set PropertyName[?Yield, ?Await] ( PropertySetParameterList ) { FunctionBody[~Yield, ~Await] }
-      if (prop.key.name === "get" || prop.key.name === "set") {
+      else if (prop.key.name === "get" || prop.key.name === "set") {
         isAccessor = true;
```

Comment:
There are 4 unnecessary member access. 2 unnecessary function call here that slow down on performance. Also 2x string comparison. No plans to optimize it?

Reply:
Thanks! Will address in another PR.